core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/store/dot-palette.store.ts

```diff
@@ -195,13 +201,29 @@ export class DotPaletteStore extends ComponentStore<DotPaletteState> {
                         lang: languageId || '1',
                         filter: filter || '',
                         offset: (event && event.first.toString()) || '0',
-                        query: `+contentType: ${this.contentTypeVarName} +deleted: false`
+                        query: `+contentType: ${
+                            this.contentTypeVarName
+                        } +deleted: false ${this.getExperimentVariantQueryField()}`.trim()
                     })
                     .pipe(take(1))
                     .subscribe((response: ESContent) => {
                         this.setLoaded();
-                        this.setTotalRecords(response.resultsSize);
-                        this.setContentlets(response.jsonObjectView.contentlets);
+                        if (this.dotSessionStorageService.getVariationId() !== DEFAULT_VARIANT_ID) {
+                            // GH issue: https://github.com/dotCMS/core/issues/26363
+                            // This is a workaround to remove the original (variant: DEFAULT) when exist a modified contentlet inside a
+                            // variant (it make a copy of the original) the endpoint return the original and the derivated/duplicated.
+                            // We need to discus about create or not a new endpoint to get the contentlets taking
+                            // in consideration the variant contentlets, if you remove this, the contentlets will show the duplicated and the original contentlet
+                            const contentlets = this.removeDerivatedCotentlets(
+                                response.jsonObjectView.contentlets
+                            );
+
+                            this.setTotalRecords(contentlets.length);
+                            this.setContentlets(contentlets);
```

Comment:
We are updating the state twice here, should we create a new updater to update the state only once?

Reply:
I removed one.